template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/appsettings.json

```diff
@@ -2,33 +2,18 @@
 ////#if (OrganizationalAuth || IndividualAuth)
 //  "Microsoft": {
 //    "AspNetCore": {
-//      "Hosting": {
-//        "Kestrel": {
-//          "Endpoints": {
-//            "Localhost": {
-//              "Address": "127.0.0.1",
-//              "Port": "55555"
-//            },
-//            "LocalhostHttps": {
-//              "Address": "127.0.0.1",
-//              "Port": "43434",
-//              "Certificate": "HTTPS"
-//            }
-//          }
-//        }
-//      },
 //#if (IndividualLocalAuth)
```

Comment:
This file has unbalanced #if & #endif statements. There are 6 #if's, but only 5 #endif's. It appears the old version in this PR has the same problem. I believe the missing #endif should be at approximately line 57 (fine-tune to get the curly brace inclusion-exclusion correct).